### PR TITLE
Prepare 1.11.0 release notes

### PR DESCRIPTION
**Pull request by ZacSweers**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+```
+
+Generates a `Taco.kts` file with the following contents:
+
+```kotlin
+println("hello!")
+```
+
+* New: Emit trailing commas for multi-line parameters and annotations.
+* New: Add support for `KSAnnotation.toAnnotationSpec()`.
```
Comment (ZacSweers) [inline docs/changelog.md:26]:
```suggestion
* New: Add `KSAnnotation.toAnnotationSpec()`.
```

File: docs/changelog.md [hunk 2]
```diff
@@ -1,6 +1,40 @@
...
+  .build()
+```
+
+Generates a `Taco.kts` file with the following contents:
+
+```kotlin
+println("hello!")
+```
+
+* New: Emit trailing commas for multi-line parameters and annotations.
```
Comment (ZacSweers) [inline docs/changelog.md:25]: Unclear if this was new or a fix 😅. CC @JakeWharton

File: docs/changelog.md [hunk 3]
```diff
@@ -1,6 +1,40 @@
...
+* New: Add `Unit` and `CharSequence` conversions in `javapoet-interop`.
+* New: Add support for default imports in `FileSpec`.
+  * This is particularly oriented at scripting support, but can also be used in non-script files.
+* New: Update to Kotlin 1.6.10.
+* Fix: Fail compilation if you only pass one string to `ClassName`.
+* Fix: Inline `val` property if its getter is `inline`.
+* Fix: Add `yield` to the list of reserved keywords.
+* Fix: Enforce only allowed parameter modifiers in `ParameterSpec` (i.e. `crossinline`, `vararg`, and `noinline`).
+* Fix: Fix `CodeBlock`s in class delegation getting `toString()`'d instead of participating in code writing.
+* Fix: Error when attempting to convert KSP error types (i.e. if `KSType.isError` is true).
```
Comment (ZacSweers) [inline docs/changelog.md:36]:
```suggestion
* Fix: Error when attempting to convert KSP error types (i.e. if `KSType.isError` is true) to `TypeName`.
```

File: docs/changelog.md (hunk 2 above)
Comment (JakeWharton) [reply docs/changelog.md:25]: New seems fine as it affects the minimum supported Kotlin language version. Not sure if it actually bumps it or not, I suspect this and square brackets for arrays in annotations are competing for the title of defining the minimum.

**Review by Egorand — APPROVED** [review]
Will merge and release after lunch

**Pull request merged**